src/content/docs/apm/agents/java-agent/configuration/java-agent-configuration-config-file.mdx

```diff
@@ -2584,7 +2584,7 @@ For more information about setting up distributed tracing, see [Enable distribut
 
 ## Error collector [#Error_Collector]
 
-These options are set in the `error_collector` stanza and unless noted otherwise can be [overridden](#System_Properties) by using a `newrelic.config.error_collector` prefixed system property. The error collector captures information about uncaught exceptions and sends them to New Relic for viewing.
+Set the error collector settings in the `error_collector` section. Unless otherwise noted, they can be [overridden](#System_Properties) with a `newrelic.config.error_collector` prefixed system property. The error collector captures information about uncaught exceptions and sends them to New Relic for viewing.
```

Comment:
```suggestion
Set the error collector settings in the `error_collector` section. Unless otherwise noted, you can [override](#System_Properties) these settings with a `newrelic.config.error_collector` prefixed system property. The error collector captures information about uncaught exceptions and sends them to New Relic for viewing.
```